### ISSUE TITLE
It dosen't work in a problem of the pintia problemset

### ISSUE THREAD

**Issue by bellongyan** [jmerle/competitive-companion#348]
When I use the extension for coding pintia exercise, it doesn't work and show an alert to me. However, it works fine with both the previous practise question and the latter one. The content of alert infomation in broswer console is that:

```js
content.js:1  TypeError: Cannot read properties of null (reading '1')
    at PTAProblemParser.parse (common.js:1:396119)
    at p (content.js:1:461)
    at content.js:1:1404
p @ content.js:1
await in p(异步)
(匿名) @ content.js:1
Promise.then(异步)
e @ content.js:1
(匿名) @ common.js:1
```

The problerm link: [https://pintia.cn/problem-sets/994805260223102976/exam/problems/994805278589960192](https://pintia.cn/problem-sets/994805260223102976/exam/problems/994805278589960192)

![Detail](https://img1.imgtp.com/2023/05/11/LOFOOvsQ.png)

**Comment by bellongyan** [comment]
It also does not work in problem: [https://pintia.cn/problem-sets/994805260223102976/exam/problems/994805275792359424](https://pintia.cn/problem-sets/994805260223102976/exam/problems/994805275792359424)

**Comment by jmerle** [comment]
Fixed in 39c0e618b14ba928daeb3c4c34d9952addd337bd, will be released within a few hours. Thanks for reporting the issue!
